### PR TITLE
Min platform version message

### PR DESCRIPTION
**Pull request by olmisa**
* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Message about min required platform version added

## Related PRs or issues

[AB#77170](https://msmobilecenter.visualstudio.com/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/77170)

### CODE REVIEW

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 1]
```diff
@@ -38,6 +39,18 @@ public void OnPreprocessBuild(BuildTarget target, string path)
         {
 #if !APPCENTER_DONT_USE_NATIVE_STARTER
             AddStartupCode(new AppCenterSettingsMakerIos());
+#endif
+        }
+        else if (target == BuildTarget.WSAPlayer)
+        {
+#if UNITY_WSA
+            const string requiredUWPVersion = "10.0.16299.0";
```
Comment (MatkovIvan) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]: Move const out of the function

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 2]
```diff
@@ -38,6 +39,18 @@ public void OnPreprocessBuild(BuildTarget target, string path)
...
+            string currentMinPlatformVersion = EditorUserBuildSettings.wsaMinUWPSDK;
```
Comment (MatkovIvan) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]: var

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs (hunk 1 above)
Comment (MatkovIvan) [reply Assets/AppCenter/Editor/AppCenterPreBuild.cs]: Store `Version` class

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 3]
```diff
@@ -38,6 +39,18 @@ public void OnPreprocessBuild(BuildTarget target, string path)
...
+            Version minRequiredUWPVersion = new Version(requiredUWPVersion);
+            if (!string.IsNullOrEmpty(currentMinPlatformVersion) && (new Version(currentMinPlatformVersion) < minRequiredUWPVersion))
+            {
+                Debug.LogWarning($"Minimum platform version should be set to {requiredUWPVersion} but it is set to {currentMinPlatformVersion}");
```
Comment (MatkovIvan) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]: > should be set

Please mention why. (App Center does not support lower versions)

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 4]
```diff
@@ -38,6 +42,16 @@ public void OnPreprocessBuild(BuildTarget target, string path)
...
             AddStartupCode(new AppCenterSettingsMakerIos());
+#endif
+        }
+        else if (target == BuildTarget.WSAPlayer)
+        {
+#if UNITY_WSA
+            var currentMinPlatformVersion = EditorUserBuildSettings.wsaMinUWPSDK;
+            if (!string.IsNullOrEmpty(currentMinPlatformVersion) && (new Version(currentMinPlatformVersion) < minRequiredUWPVersion))
+            {
+                Debug.LogWarning($"Minimum platform version should be set to {minRequiredUWPVersion} because of App Center does not support lower versions but it is set to {currentMinPlatformVersion}");
```
Comment (annakocheshkova) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]:
```suggestion
                Debug.LogWarning($"Minimum platform version should be set to {minRequiredUWPVersion. App Center does not support lower versions but it is set to {currentMinPlatformVersion}");
```

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 5]
```diff
@@ -11,6 +12,9 @@
 public class AppCenterPreBuild : IPreprocessBuildWithReport
 {
     public int callbackOrder { get { return 0; } }
+#if UNITY_WSA
+    private readonly Version minRequiredUWPVersion = new Version("10.0.16299.0");
```
Comment (MatkovIvan) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]:
```suggestion
    private readonly Version RequiredMinimalUWPVersion = new Version("10.0.16299.0");
```

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs [hunk 6]
```diff
@@ -38,6 +42,16 @@ public void OnPreprocessBuild(BuildTarget target, string path)
         {
 #if !APPCENTER_DONT_USE_NATIVE_STARTER
             AddStartupCode(new AppCenterSettingsMakerIos());
+#endif
+        }
+        else if (target == BuildTarget.WSAPlayer)
+        {
+#if UNITY_WSA
+            var currentMinPlatformVersion = EditorUserBuildSettings.wsaMinUWPSDK;
```
Comment (MatkovIvan) [inline Assets/AppCenter/Editor/AppCenterPreBuild.cs]:
```suggestion
            var currentMinPlatformVersion = new Version(EditorUserBuildSettings.wsaMinUWPSDK);
```

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs (hunk 4 above)
Comment (MatkovIvan) [reply Assets/AppCenter/Editor/AppCenterPreBuild.cs]: > should be set to {minRequiredUWPVersion}

or higher

**Review by MatkovIvan — CHANGES_REQUESTED** [review]

File: Assets/AppCenter/Editor/AppCenterPreBuild.cs (hunk 4 above)
Comment (olmisa) [reply Assets/AppCenter/Editor/AppCenterPreBuild.cs]: Fixed

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**